PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -296,23 +301,30 @@ local function CreateCamera()
 	end
 
 	function this:ApplyVRPosition()
```

Comment:
Since we do more than apply just position we should name this function ApplyVRTransformation